setup.cfg

```diff
@@ -31,18 +31,44 @@ auto_use = True
 [metadata]
 package_name = sunpy
 description = SunPy: Python for Solar Physics
+long_descripton = An open-source Python library for Solar Physics data analysis and visualization.
 author = The SunPy Community
 author_email = sunpy@googlegroups.com
-license = BSD
-url = http://sunpy.org/
+license = BSD 2-Clause
+url = https://sunpy.org
 edit_on_github = True
 github_project = sunpy/sunpy
-
-[flake8]
-max-line-length = 100
+install_requires = numpy>=1.11, astropy>=2.0.3>=3.0.2, scipy, pandas>=0.12.0, matplotlib>=1.3
```

Comment:
One day be nice to either parse the reqs.txt or define it all here.

Reply:
Do we want them anymore?

Reply:
I thought RTD uses them at least?

Reply:
yes it does

Reply:
But maybe we should jsut dump it all into the yaml for RTD and delete those.